public/js/invertedIndex.js

```diff
@@ -0,0 +1,155 @@
+/**
```

Comment:
Your inverted index class should only create, search and get indexes. Move all helper methods like tokenize to another file and call the methods within the inverted class